### ISSUE TITLE
Exclude donations under $5 from GIVbacks eligibility in the API

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-dapps-v2#4639]
And let make it possible that we can change the limit number later

cc @WhyldWanderer

**Comment by laurenluz** [comment]
Actually @mohammadranjbarz let's make the minimum $4 :)

**Comment by laurenluz** [comment]
Ok one more thing... let's make also every donation, no matter the size, that goes to the Giveth community of makers project eligible for GIVbacks.

**Comment by mohammadranjbarz** [comment]
@maryjaf  Can you test this?
If you have any question feel free to DM me

**Comment by maryjaf** [comment]
When I set a value for this new field "minEligibleValueUsd", in GIVabck api, the response is changed correctly based on minimum usd value

![image](https://github.com/user-attachments/assets/675dbb67-49e5-4f38-be9a-60700c7db059)

**Comment by laurenluz** [comment]
EPIC @WhyldWanderer FYI

**Comment by laurenluz** [comment]
@maryjaf did you test also this?

> Ok one more thing... let's make also every donation, no matter the size, that goes to the Giveth community of makers project eligible for GIVbacks.

when we set minUSDvalue to $4... it should show all donations above $4 PLUS all the donations to the giveth community of makers project. on prod, that's https://giveth.io/donate/the-giveth-community-of-makers

**Comment by maryjaf** [comment]
> @maryjaf did you test also this?
> 
> > Ok one more thing... let's make also every donation, no matter the size, that goes to the Giveth community of makers project eligible for GIVbacks.
> 
> when we set minUSDvalue to $4... it should show all donations above $4 PLUS all the donations to the giveth community of makers project. on prod, that's https://giveth.io/donate/the-giveth-community-of-makers

Yeah @laurenluz  by setting project slug in this new field "givethCommunityProjectSlug", all eligible donation to this project is shown even thought the amount is lower than minimum 

![image](https://github.com/user-attachments/assets/226d9276-975d-4281-8a90-e0c3182a62bf)

**Comment by maryjaf** [comment]
> It works as expected, when I set a value for this new field "minEligibleValueUsd", in GIVabck api, the response is changed correctly based on minimum usd value
> 
> ![image](https://private-user-images.githubusercontent.com/111529185/363997269-675dbb67-49e5-4f38-be9a-60700c7db059.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU4OTc3MDIsIm5iZiI6MTcyNTg5NzQwMiwicGF0aCI6Ii8xMTE1MjkxODUvMzYzOTk3MjY5LTY3NWRiYjY3LTQ5ZTUtNGYzOC1iZTlhLTYwNzAwYzdkYjA1OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwOVQxNTU2NDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOWYzZDFjMjM1ZjIwNTUyMzk2OGIwNDAzOTcyZmQ1NDQzNjdlYTBkZGY4MTI4YzE5ZTUzZGEwNmI3MGZhNmNkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.LIP1VlkNekUAbWwksPpJIWNjV7aWEPJFotGBoZQsE_g)

after the call I doubt about this issue based on the title of this issue and the test result that I reported in above comment 
when the new field in GIVabck api ,"minEligibleValueUsd" is set for example 5$
in the response only the donations bigger or equal  than 5 is  shown in response of gibveck eligible api 

but in the meeting this subject is discussed that it is cumulative 5 $
![image](https://github.com/user-attachments/assets/c6af71ba-d00f-4a6e-b89d-86c43a1cc4ab)

@mohammadranjbarz @laurenluz

**Comment by laurenluz** [comment]
oh sorry @maryjaf I made a mistake writing in the notes, I was doing it too quick. it's single donation to a single project, not cumulative. I updated the side answer in there now.

the way you tested it before was correct. 

so sorry for the confusion!

**Comment by WhyldWanderer** [comment]
@mohammadranjbarz @maryjaf - I am currently trying to make the spreadsheet for Round 73 GIVbacks and when I use the new eligible-donations script, it is still including donations under 4$

Here is the parameters I used:
![image](https://github.com/user-attachments/assets/1a7ac41f-c23b-4b38-829e-c3c5a57c4468)

Here is the data I got:
https://docs.google.com/spreadsheets/d/1fE6d3wAL_8dfA73ZdcFzd_nUCtv8Fv0inQqHRabqWh0/edit?usp=sharing

I tried with $4 and 0$ both so that I could compare the two. You will see the tabs at the bottom labeled accordingly. Am I entering something incorrectly in the API?
